pullrequest.rst

```diff
@@ -76,111 +159,67 @@ Fifth, proper :ref:`documentation <documenting>`
 additions/changes should be included.
 
 
-.. _patch-generation:
-
-Generation
-''''''''''
-
-To perform a quick sanity check on your changes, you can run::
-
-   make patchcheck
-
-This will check and/or fix various common things people forget to do for
-pull requests, such as adding any new files needed for the pull request to work
-(note that not all checks apply to non-core developers).  On Windows, use this
-command (after any successful build of Python)::
-
-   python.bat Tools/scripts/patchcheck.py
```

Comment:
This has been combined with the other patchcheck section and has been moved further down.